relayers.json

```diff
@@ -39,7 +39,15 @@
     {
         "name": "OpenRelay",
         "url": "https://openrelay.xyz/",
+        "logo": "openrelay.png",
         "networks": [
+            {
+                "networkId": 1,
+                "sra_http_endpoint": "https://api.openrelay.xyz/v0/",
+                "static_order_fields": {
+                    "fee_recipient_addresses": ["0xc22d5b2951db72b44cfb8089bb8cd374a3c354ea"]
+                }
+            },
             {
```

Comment:
Does OpenRelay not have a Ropsten network orderbook? If not, can you remove the Ropsten (network 3) entry in this array as well?